### ISSUE TITLE
[Feature] Legacy authentication

### ISSUE THREAD

**Issue by GioF71** [tamland/airsonic-refix#138]
Hello, I am trying to install you application on top of Lightweight Media Server, but it cannot authenticate.
I get this error:

```
lms-lightweight-music-server | [2024-May-12 14:01:49.815] 1 - [error] - 140300623276856 [API_SUBSONIC] Error while processing request '/ping', params = [{c=app}, {f=json}, {s=xGYQ8Hmt5hNr3dbG}, {t=6f2bc816c4c77a99cf86da9b3d8c1710}, {u=giovanni}, {v=1.15.0}, ], code = 41, msg = 'Token authentication not supported for LDAP users.'
lms-lightweight-music-server | [2024-May-12 14:01:49.815] 1 - [info] "wthttp: 192.168.1.14   GET /rest/ping?u=giovanni&s=xGYQ8Hmt5hNr3dbG&t=6f2bc816c4c77a99cf86da9b3d8c1710&v=1.15.0&c=app&f=json HTTP/1.1 200 196"
lms-lightweight-music-server | [2024-May-12 14:01:49.815] 1 - [info] "WebRequest: took 0.642 ms"
lms-lightweight-music-server | [2024-May-12 14:01:49.874] 1 - [info] "wthttp: 192.168.1.14   GET /rest/ping?u=giovanni&p=HEn%25CWgnRdfz5iAt2%24495zcE&v=1.15.0&c=app&f=json HTTP/1.1 200 109"
lms-lightweight-music-server | [2024-May-12 14:01:49.874] 1 - [info] "WebRequest: took 16.946 ms"
```

From my direct experience, using the py-sonic library (so not the same thing you use), the only thing I need to change for lms is "legacy authentication". See [this](https://github.com/epoupon/lms/issues/362) issue for a reference.
Would it be possible to allow this authentication mode so that lms can be supported?

Thank you!

**Comment by tamland** [comment]
This should be supported. As you can se from your log there is a second call using legacy auth: `GET /rest/ping?u=giovanni&p=HEn%25CWgnRdfz5iAt2%24495zcE&v=1.15.0&c=app&f=json `

**Comment by GioF71** [comment]
You are totally right, the authentication infact goes through as you say. But anyway I get an error in the interface saying:

Could not log in. (Failed to fetch)

That lead me to believe that the problem was with the authentication. Now I understand that the app first tries with normal authentication only to fallback to legacy in case of failure.

I enabled debug log on lms but there are no new information telling me what could be failing after the authentication.
Can the log level be raised on Airsonic-refix?

Thank you!

**Comment by GioF71** [comment]
Hello, I just wanted to let you know that I could make it work.
I am using a reverse proxy (traefik) on both airsonic-refix and lms, and had to add accessControlAllowOriginList to the traefik middleware settings for lms.
Authentication already works in "legacy mode" as you were saying.
Thank you!
